framework/lib/components/file-picker/types.ts

```diff
@@ -50,6 +50,7 @@ export interface FilePickerProps {
   maxFileSize?: number
   value?: string | null
   loadWithSpinner?: boolean
+  variant?: 'ai'
```

Comment:
Do we want to default the variant to ai? Same for below in this file

Reply:
No,  the default is undefined which in that case is just baseStyle.